addons/map_gestures/functions/fnc_getProximityPlayers.sqf

```diff
@@ -37,7 +37,7 @@ switch (_this) do {
         _proximityPlayers append allUnits;
     };
     case (1): { // Players in Group
-        _proximityPlayers append group ace_player;
+        _proximityPlayers append units group ace_player;
```

Comment:
```suggestion
        _proximityPlayers append (units group ace_player);
```

Reply:
`units` works also with just the unit.
`_proximityPlayers append (units ace_player);`

Reply:
> `units` works also with just the unit.
> `_proximityPlayers append (units ace_player);`

Sounds good.